lib/jenkins_pipeline/job.rb

```diff
@@ -31,7 +31,7 @@ def serialize
     private
 
     def build_status
-      return "unknown" unless @upstream_status == "success"
+      return "unknown" if %w(unknown failure aborted running).include?(@upstream_status)
```

Comment:
In this case, I would prefer to have a separate method to handle the @upstream_status logic with something like: `"success" || ""` what do you think?  
Also, is there a way to add a test for this case?